mavros/include/mavros/utils.h

```diff
@@ -98,5 +100,38 @@ int sensor_orientation_from_str(const std::string &sensor_orientation);
  */
 timesync_mode timesync_mode_from_str(const std::string &mode);
 
+/**
+ * @brief Conversion from height above geoid (AMSL)
+ * to height above ellipsoid (WGS-84)
+ */
+template <class T>
+inline double geoid_to_ellipsoid_height(T lla){
+	try {
+		GeographicLib::Geoid egm96("egm96-5");
```

Comment:
Please check how expencive creation of `Geoid` class. If it loads data each time - that should be changed.